go.mod

```diff
@@ -47,3 +45,5 @@ require (
 	google.golang.org/grpc v1.64.1 // indirect
 	google.golang.org/protobuf v1.34.2 // indirect
 )
+
+replace github.com/wasmCloud/provider-sdk-go => ../../wasmCloud/provider-sdk-go
```

Comment:
Note to self: remove this in favor of updating the provider SDK after https://github.com/wasmCloud/provider-sdk-go/pull/24 lands.